easybuild/easyconfigs/i/intel-compilers/intel-compilers-2021.3.0.eb

```diff
@@ -26,10 +23,10 @@ checksums = [
     'c4553f7e707be8e8e196f625e4e7fbc8eff5474f64ab85fc7146b5ed53ebc87c',  # l_fortran-compiler_p_2021.3.0.3168_offline.sh
 ]
 
-local_gccver = '10.3.0'
+local_gccver = '11.2.0'
```

Comment:
@robert-mijakovic Heads-up: we're bumping this because the plan is to use `GCC` 11.2 as a base for `foss/2021b` and `intel/2021b`